Mathlib/Data/Fin/Basic.lean

```diff
@@ -986,6 +986,8 @@ theorem lt_add_one_iff {n : ℕ} {k : Fin (n + 1)} : k < k + 1 ↔ k < last n :=
   simp
 #align fin.lt_add_one_iff Fin.lt_add_one_iff
 
+attribute [nolint simpNF] lt_add_one_iff add_one_lt_iff -- HACK: CovariantClass lemma times out sigh
+
```

Comment:
I guess we better test this again, but I'm happy if this PR goes through before we get to that.